Formula/opensearch.rb

```diff
@@ -0,0 +1,123 @@
+class Opensearch < Formula
+  desc "Open source distributed and RESTful search engine"
+  homepage "https://github.com/opensearch-project/OpenSearch"
+  url "https://github.com/opensearch-project/OpenSearch/archive/refs/tags/1.0.0-beta1.tar.gz"
+  sha256 "d23385aa42f636049ae270bdb496843dc8d2dfd88bd7f4761e305e8193b76399"
+  license "Apache-2.0"
+
+  depends_on "gradle@6" => :build
+  depends_on "openjdk"
+
+  def cluster_name
+    "opensearch_#{ENV["USER"]}"
+  end
```

Comment:
wouldn't this always be `opensearch_brew` because CI?

Reply:
Technically, guess so. Not when built from source, though. That naming seems fine anyway though, right?

Reply:
yeah, but in that case it hardly needs to be a method IYAM.